astropy/units/imperial.py

```diff
@@ -2,8 +2,15 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
 """
-This package defines colloquially used Imperial units.  By default, they
-are not enabled.  To enable them, do::
+This package defines colloquially used Imperial units.  They are
```

Comment:
Do they actually *replace* normal units,  e.g. does `u.eV` get redefined?  I thought they still need to be explicitly used as `u.cds.<unit>`.